app/domain/mailing_list/subscribers.rb

```diff
@@ -116,4 +109,25 @@ def event_subscribers(condition)
                  Event.sti_name,
                  true)
   end
+
+  private
+
+  def excluded_people_by_tag
```

Comment:
since you're returning a list of ids, it makes sense to include keyword ids in method name.

tag_excluded_person_ids ?